lib/OpenCloud/ObjectStore/Resource/AbstractResource.php

```diff
@@ -36,7 +38,7 @@
     /** @var string The FQCN of the metadata object used for the container. */
     protected $metadataClass = 'OpenCloud\\Common\\Metadata';
 
-    /** @var \OpenCloud\Common\Service\ServiceInterface The service object. */
```

Comment:
How is this not valid? It's an interface which exists. I'd prefer to associate types with their interfaces instead of concrete classes.